website/docs/r/compute_network_endpoint.html.markdown

```diff
@@ -103,7 +103,7 @@ The following arguments are supported:
   The instance must be in the same zone of network endpoint group.
 
 * `port` -
-  (Required)
+  (Optional)
   Port number of network endpoint.
```

Comment:
This should also include ([from the API docs](https://cloud.google.com/compute/docs/reference/rest/beta/networkEndpointGroups/attachNetworkEndpoints)): `If not specified, the defaultPort for the network endpoint group will be used.`

Reply:
Here's the docs I refer to https://cloud.google.com/load-balancing/docs/negs/setting-up-zonal-negs

Reply:
In addition to the change specified above, this docs file also need to be updated to specify the new possible value of GCE_VM_IP.